### ISSUE TITLE
Microsoft Remote Desktop download link

### ISSUE THREAD

**Issue by grahampugh** [autopkg/rtrouton-recipes#116]
I notice that the fwlink for the MRD recipe is currently pointing to an older version than the current. From a Slack search, it seems the link has to be manually updated and so can get out of date. 

This seems to have prompted a change to the [macadmins.software](https://macadmins.software/latest.xml), which now has the `download` link as a direct Akamai link with a separate key for `fwlink`. 

Perhaps (and only perhaps) it's worth updating the recipe to point to the direct link rather than using the `fwlink`?

**Comment by rtrouton** [comment]
The direct download link appears to be pointing to a specific version:

`https://officecdn-microsoft-com.akamaized.net/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/Microsoft_Remote_Desktop_10.6.1_installer.pkg`

The problem I foresee is that once the next MRD version drops, this link will also be out of date and I'll have to revisit the recipe to edit it again. If you have a solution for this issue, I'd be interested in it.

**Comment by grahampugh** [comment]
Yes, it depends if that link is being maintained better than the fwlink :) Probably worth leaving and keeping an eye on, but a processor to scrape the XML would be the only sane replacement, I guess.

**Comment by grahampugh** [comment]
Thanks to your recipe design, I've been able to temporarily override the URL by adding `--key DOWNLOADURL=https://officecdn-microsoft-com.akamaized.net/pr/C1297A47-86C4-4C1F-97FA-950631F94777/MacAutoupdate/Microsoft_Remote_Desktop_10.6.1_installer.pkg`
